packages/@aws-cdk/aws-codepipeline/package.json

```diff
@@ -100,6 +99,7 @@
     "@aws-cdk/aws-kms": "0.0.0",
     "@aws-cdk/aws-s3": "0.0.0",
     "@aws-cdk/core": "0.0.0",
+    "@aws-cdk/cx-api": "0.0.0",
```

Comment:
Doesn't this need to be in `peerDependencies` now too?

Reply:
Added